components/FrontPage/Combined/WriteOnceAccessAnywhere.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import TBox from "../textbox";
+import Btn from "../button";
+import Image from "next/image";
+
+// This is the 2nd to last section with the "write, once access anywhere" catchphrase
+
+export default function WriteOnceAccessAnywhere() {
+  return (
+    <div className="flex h-[700px] w-screen justify-center bg-slate-200 dark:bg-slate-800">
+      <div className="flex h-full max-w-[1200px] items-center justify-center gap-24 px-6">
+        <div className="ml-3 max-w-[90%] sm:max-w-[50%]">
+          {" "}
+          <TBox variant="h2">write once,</TBox>
+          <TBox variant="h2">access anywhere</TBox>
```

Comment:
We can make this less crowded.
```suggestion
          <TBox variant="h2">write once, <br />access anywhere</TBox>
```